### PR TITLE
feat: mocks upload files in tests

### PR DESCRIPTION
**Pull request by ReidyT**
Upload the image before the tests.
This PR propose a solution to the PR #136. The apps-query-client from [graasp/graasp-apps-query-client#](https://github.com/graasp/graasp-apps-query-client/pull/389#event-13911780444) should be used, because MSW is mocking the upload and download of app settings files.

### CODE REVIEW

**Review by LinaYahya — APPROVED** [review]
Great work @ReidyT

File: cypress/fixtures/images/utils.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+export const createFileInstance = (base64Data: string): File => {
+  const binaryString = window.atob(base64Data);
+  const byteNumbers = new Uint8Array(binaryString.length);
+  for (let i = 0; i < binaryString.length; i += 1) {
+    byteNumbers[i] = binaryString.charCodeAt(i);
+  }
+  return new File([new Blob([byteNumbers])], '');
+};
+
```
[the remark this answers is not in the document]
Comment (ReidyT) [reply cypress/fixtures/images/utils.ts:11]: I was not able to load a file by its path, so I decided to create a File from a base64 file. Do you know a better solution for that?

**Review by pyphilia — APPROVED** [review]

File: cypress/fixtures/images/utils.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (ReidyT) [reply cypress/fixtures/images/utils.ts]: I wasn't able to load the file correctly.

**Comment by ReidyT** [comment]
There is an issue in the builder view where we have to click twice on the "Add the labels" step. I created a separated issue for that, because it was not introduced in this PR. https://github.com/graasp/graasp-app-name-the-frame/issues/167

**Pull request merged**